cmd/zed/README.md

```diff
@@ -11,7 +21,7 @@ For all `zq` options, use the help subcommand:
 zq help
 ```
 
-## Examples
+### Examples
 
 Here are a few examples using a small Zeek formatted log file, `conn.log`,
 located in this directory. See the
```

Comment:
I know we're due to rewrite examples using non-Zeek data, but if we want to keep this ref here for now, we should move `conn.log` from `cmd/zq` into this directory so the "in this directory" reference is accurate.